tools/sklearn/train_test_eval.py

```diff
@@ -395,23 +420,25 @@ def main(inputs, infile_estimator, infile1, infile2,
         if isinstance(estimator, pipeline.Pipeline):
```

Comment:
@kxk302 pipeline does not seem to be defined? At least the linter complains and this seems to be serious.

Reply:
@bgruening Hmm, I had started on fixing some of the lint issues and it seems I did not push some of the files. Pulled from upstream and pushed them just now with revised files. When tests are green, it should be good to go. Thanks.